### DISCUSSION TITLE
How to remove the title bar

### DISCUSSION THREAD

**Discussion by dingwalnitin** [elementary/triage#331] [Q&A]
Hi folks, how do i remove the title bar, its very annoying, currently using edge and it occupies major portion of the screen and is very annoying visually.
![image](https://github.com/elementary/triage/assets/90894005/7459fc39-ba79-42e1-b8a6-606455b29f26)

the white title bar named New Discussion ..........................

thanks for support

**Comment by IanHead** [comment]
You can right-click in the tab bar in Edge and turn off `Use system title bar and borders`.

**Reply by dingwalnitin** [reply]
i am not getting any such option

**Reply by Ditract** [reply]
It's as the previous response says; you should right-click and choose the option to remove borders.
